ex16v2_ch4.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state constant
+LEGAL_DRIVING_AGE = 16
+
+#step2: get input from user
+age = int(input("Enter your age: "))
+
+#step3: set up 3 conditionals: if user provides invalid input, if 16 or older, or if younger than 16
+if age < 0:
+    print("Please enter a valid age.")
+
+elif age >= 16:
```

Comment:
To take this to the next level, I'd personally keep the valid age check separate from the legal driving age check as they are separate concepts or concerns. So something like this:

```
# Check valid age
if age < 0:
  ## invalid age

#  Check whether the person can drive
if age >= 16:
  ##
else:
 ##
```